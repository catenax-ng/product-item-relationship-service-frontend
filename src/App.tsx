import { Provider } from "react-redux";
import { RouterProvider } from "react-router-dom";
import { PersistGate } from "redux-persist/integration/react";
import { KeyCloakProvider } from "./lib/keycloak";
import { router } from "./pages/Router";
import { persistor, store } from "./store/store";

function App() {
  return (
    <Provider store={store}>
      <KeyCloakProvider>
        <PersistGate loading={null} persistor={persistor}>
          <RouterProvider router={router} />
        </PersistGate>
      </KeyCloakProvider>
    </Provider>
  );
}

export default App;
