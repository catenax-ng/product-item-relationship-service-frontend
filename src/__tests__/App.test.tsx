import App from "../App";
import { render } from "../testing/test-utils";
it("renders the App ", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
