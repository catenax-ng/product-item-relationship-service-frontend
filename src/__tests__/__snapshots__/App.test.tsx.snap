// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the App  1`] = `
<div>
  <div>
    <header
      class="css-on3ivn"
    >
      <div
        class="MuiBox-root css-1becb2a"
      >
        <div
          class="MuiBox-root css-429xfk"
        >
          <img
            class="MuiBox-root css-1emgxcc"
            src="/img/cx-logo-text.svg"
          />
        </div>
        <div
          class="MuiBox-root css-14tu3sa"
        >
          <nav
            class="MuiBox-root css-2imjyh"
          />
        </div>
        <div
          class="MuiBox-root css-19g4w2v"
        >
          <div
            class="MuiBox-root css-0"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall MuiButton-containedSizeSmall css-73kr90-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Help
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </header>
    <div
      class="MuiBox-root css-5kk6ic"
    >
      <div
        class="MuiBox-root css-1ybz6by"
      >
        <img
          alt="home stage"
          class="object-fit x-left-40"
          src="./img/home-stage-desktop.png"
          style="bottom: 0px; height: 100%; left: 0px; object-fit: cover; right: 0px; top: 0px; visibility: visible; width: 100%;"
        />
      </div>
      <div
        class="MuiBox-root css-xownll"
      >
        <div
          class="children-header"
        >
          <div
            class="css-180vpgq"
          >
            <img
              class="MuiBox-root css-1baulvz"
              src="/img/IRS_FE_Logo_long.png"
            />
          </div>
          <div
            class="css-180vpgq"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Insights into decentral stored Data Chains
            </p>
          </div>
          <section
            class="css-jlkg07"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-animated css-y7y8hp-MuiFormLabel-root-MuiInputLabel-root"
            >
              content.irs.form.environment.label
            </label>
            <div
              class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth  css-12005ah-MuiInputBase-root-MuiFilledInput-root-MuiSelect-root"
            >
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                class="MuiSelect-select MuiSelect-filled MuiInputBase-input MuiFilledInput-input css-d9oaum-MuiSelect-select-MuiInputBase-input-MuiFilledInput-input"
                role="button"
                tabindex="0"
              >
                DEV
              </div>
              <input
                aria-hidden="true"
                class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                tabindex="-1"
                value="DEV"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconFilled css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
                data-testid="ArrowDropDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
          </section>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
            style="margin-top: 75px;"
            tabindex="0"
            type="button"
          >
            Login
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
